walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java

```diff
@@ -37,7 +37,6 @@
   @Override
```

Comment:
there's still some diffs in this file. it might be easier just to copy and paste the original file: https://github.com/ERetana2/STEP/blob/master/walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java